### ISSUE TITLE
Output Option(s)

### ISSUE THREAD

**Issue by jessebutryn** [keepcosmos/terjira#54]
Can you please add an option to remove all the unnecessary formatting from all output?  It would be useful to have only text output that can be more easily parsed.  Currently I have to hack the output with the following:

Remove ASCII Table:

`jira issue jql 'search' | awk '/(.*)-([0-9]+)/{print $0}' | tr -d '│'`

Remove colors:

`perl -pe 's/\x1b\[[0-9;]*[a-zA-Z]//g`

This also makes me lose the line that describes the "Issue Type" however the tables and colors make it very difficult to wrap tools around this program in order to process output/generate reports etc.  

Great tool otherwise though, thank you!

**Comment by buguibu** [comment]
Let me suggest an option: include or return only the URL on the issues returned bu the query.

**Comment by unphased** [comment]
Yes i hope we can have it automatically (if output is a pipe or script, not a terminal) to produce unix-pipe friendly tabulated output, especially for stuff like `jira issue jql` and `jira issue search`.
